### PR TITLE
test(ngSanitize): disable a failing Edge test in all versions, including 18

### PR DESCRIPTION
**Pull request by mgol**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Test fix.

**What is the current behavior? (You can also link to an open issue here)**

Edge 18 fails tests.

**What is the new behavior (if this is a feature change)?**

Edge 18 passes tests (or at least the one I fixed).

**Does this PR introduce a breaking change?**

No.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- ~~Fix/Feature: [Docs](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#documentation) have been added/updated~~
- ~~Fix/Feature: Tests have been added; existing tests pass~~

**Other information**:

The test has been only disabled on Edge 16/17 so far which made it fail in Edge 18.

### CODE REVIEW

File: test/ngSanitize/sanitizeSpec.js [hunk 1]
```diff
@@ -261,8 +261,8 @@ describe('HTML', function() {
       });
     });
 
-    if (!/Edge\/(16|17)/.test(window.navigator.userAgent)) {
-      // Skip test on Edge 16 due to browser bug.
+    if (!/Edge\//.test(window.navigator.userAgent)) {
```
Comment (jbedard) [inline test/ngSanitize/sanitizeSpec.js]: Could add `\d` or `\d{2,}` to make it a little closer to the previous? 🤷‍♂️

**Review by jbedard — APPROVED** [review]

File: test/ngSanitize/sanitizeSpec.js (hunk 1 above)
Comment (mgol) [reply test/ngSanitize/sanitizeSpec.js]: You can now make such minor code suggestions directly from GitHub. :) The icon with sheets of paper & plus/minus on the front one.
It looks like that:
```suggestion
    if (!/Edge\/\d{2,}/.test(window.navigator.userAgent)) {
```

Comment (mgol) [reply test/ngSanitize/sanitizeSpec.js]: And then I can apply the suggestion with one button.

Comment (petebacondarwin) [reply test/ngSanitize/sanitizeSpec.js]: Although I heard it messes with the commit ownership?

Comment (petebacondarwin) [reply test/ngSanitize/sanitizeSpec.js]: As long as you don't use the github merge button for squashing, I think it is OK.

**Pull request merged**